### ISSUE TITLE
Bug: Recording window expanding or contracting when starting to record (Windows OS)

### ISSUE THREAD

**Issue by prof-nobody** [kevglass/floom#4]
I am noticing that the recording window will expand or contract occasionally. Normally with a change to a different resolution. If you stop recording it is fine the next time you Start Recording. I am guessing because is pulling old settings that don't get updated when it is moved to a new screen. Will test if resizing the window when moving it fixing the issue.

**Comment by prof-nobody** [comment]
Attempting to resize the window after moving it to a new screen didn't help it.

**Comment by kevglass** [comment]
What do we mean by recording window in this case?

**Comment by prof-nobody** [comment]
This would be the Indicator window / frame around the contents. I have attached outputs from the recording. it looks now like it is something UI based. I also did a gifcap capture of what is happening. All are below. 

![Recording 2023-01-04 at 13 44 54](https://user-images.githubusercontent.com/11951094/210627289-405fe051-4c4b-4349-8477-2c3deb752ac1.gif)
[window_enlarging.webm](https://user-images.githubusercontent.com/11951094/210627308-aae49b41-ff2b-4780-9f07-ffe2a1bbceb1.webm)
[window_not_enlarged.webm](https://user-images.githubusercontent.com/11951094/210627319-b00f8ad5-7ada-4d74-83a3-e5332a94cdc3.webm)
